python/ray/data/_internal/block_list.py

```diff
@@ -178,3 +178,22 @@ def executed_num_blocks(self) -> int:
         doesn't know how many blocks will be produced until tasks finish.
         """
         return len(self.get_blocks())
+
+    def randomize_block_order(self, seed: Optional[int] = None) -> None:
```

Comment:
Nit: `None` return is implied if no type annotation.
```suggestion
    def randomize_block_order(self, seed: Optional[int] = None):
```